modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCFontImpl.java

```diff
@@ -125,10 +125,24 @@ private FontStrike getFontStrike()
         return getFontStrike().getMetrics().getXHeight();
     }
 
+    private static boolean needsTextLayout(final int glyphs[]) {
+        for (int g : glyphs) {
+            if (g != 0) {
```

Comment:
So if *any* glyph is present, you don't use layout.
I was expecting the other way round, if any glyph is missing you want to try layout ..

Reply:
I presume it was intentional, given the comment at the usage site, but it seems better to do as Phil suggests.

Reply:
Sure. I will change according to @prrace's suggestion.